debian/clearwater-socket-factory.prerm

```diff
@@ -52,6 +52,12 @@ set -e
 
 case "$1" in
     remove|upgrade|deconfigure)
+        # Stop and deconfigure the management and signaling upstart jobs
+        stop clearwater-socket-factory-mgmt > /dev/null 2>&1 || true
+        stop clearwater-socket-factory-sig > /dev/null 2>&1 || true
+        rm -f /etc/init/clearwater-socket-factory-mgmt.conf
+        rm -f /etc/init/clearwater-socket-factory-sig.conf
```

Comment:
These files are controlled by the package manager and it will remove them if needed.